ARTICLE-2.markdown

```diff
@@ -12,7 +12,7 @@ A Membership shall immediately terminate upon the death of a Member.
 ### Section 2.3  Expulsion
 
 (a)	A Member may for any lawful reason be expelled from the
-        Corporation by unanimous consensus of the Member Body.
+        Cooperative by unanimous consensus of the Member Body.
```

Comment:
Agreed. And that's the idea. It should be extremely difficult to expel a member. Expulsion is a very serious situation, and should require all members to agree.

It requires all members to vote in a new member, and it requires all members to vote out a member.

However, when we get to the size where we're talking about hundreds of members, we can review this and adjust as necessary.